test/common.jl

```diff
@@ -11,6 +11,20 @@ const DETATOL = 1e-3 * D * TRATIO
 # Random tolerance
 const RNDATOL = 5e-2 * D * TRATIO * 2
 
+# Convenience
+using Distributions: Distributions
+using Bijectors: Bijectors
+struct LogDensityDistribution{D<:Distributions.Distribution}
```

Comment:
Yeaaah I was also thinking this would be nice to have somewhere it's more widely available.

But is AHMC really the right place? :confused:

Reply:
Since it depends on Bijectors, maybe it could be defined in Bijectors?

Reply:
I'm a bit uncertain if it's worth including since it's of little practical usage (beyond testing samplers). For now I've added a comment for future reference, but leaving it as is for now.